pkgs/by-name/vc/vcpkg/package.nix

```diff
@@ -32,6 +32,10 @@ stdenvNoCC.mkDerivation (finalAttrs: {
   installPhase = ''
       runHook preInstall
 
+      # If we don’t turn this off, then you won’t be able to run binaries that
+      # are installed by vcpkg.
+      find triplets -name '*linux*.cmake' -exec bash -c 'echo "set(VCPKG_FIXUP_ELF_RPATH OFF)" >> "$1"' -- {} \;
```

Comment:
Please use X_VCPKG_RPATH_KEEP_SYSTEM_PATHS, as vcpkg need to patch rpath for dependencies with dynamic libraries installed in the installation prefix.

Reply:
OK, done.